recipes/pmx_biobb/meta.yaml

```diff
@@ -24,13 +24,13 @@ requirements:
     - {{ compiler('fortran') }}
     - make
   host:
-    - python
+    - python =3.7
```

Comment:
Why is this needed, its not recommended to pin a python version.

Reply:
Hi,

This is the only workaround I found to create a package compatible with Python versions from 3.7 to 3.11. In this case

----Dependency----------------Python ver-------------Anaconda package created

- python [no pinned vers] ------3.8-----> [linux-64/pmx_biobb-4.1.2-py38h0928705_2.tar.bz2](https://anaconda.org/bioconda/pmx_biobb/4.1.2/download/linux-64/pmx_biobb-4.1.2-py38h0928705_2.tar.bz2)
- python [no pinned vers] ------3.9-----> [linux-64/pmx_biobb-4.1.2-py39hef94ad9_2.tar.bz2](https://anaconda.org/bioconda/pmx_biobb/4.1.2/download/linux-64/pmx_biobb-4.1.2-py39hef94ad9_2.tar.bz2)
- python [no pinned vers] ------3.10----> [linux-64/pmx_biobb-4.1.2-py310h95c5ba8_2.tar.bz2](https://anaconda.org/bioconda/pmx_biobb/4.1.2/download/linux-64/pmx_biobb-4.1.2-py310h95c5ba8_2.tar.bz2)

- python =3.7 ------------------3.7-----> [linux-64/pmx_biobb-4.1.2-py37h8537716_1.tar.bz2](https://anaconda.org/bioconda/pmx_biobb/4.1.2/download/linux-64/pmx_biobb-4.1.2-py37h8537716_1.tar.bz2)
- python >=3.7 -----------------3.11-----> [linux-64/pmx_biobb-4.1.2-py311h8537716_0.tar.bz2](https://anaconda.org/bioconda/pmx_biobb/4.1.2/download/linux-64/pmx_biobb-4.1.2-py311h8537716_0.tar.bz2)

Is there any other "official/recommended" way to force bioconda to create the 5 packages, not using pining trick?